pipes/WDL/tasks/tasks_nextstrain.wdl

```diff
@@ -85,9 +147,9 @@ task augur_mafft_align {
             --reference-sequence ~{ref_fasta} \
             --output ~{basename}_aligned.fasta \
             ~{true="--fill-gaps" false="" fill_gaps} \
-            ~{true="--existing-alignment " false="" existing_alignment} \
+            ~{"--existing-alignment " + existing_alignment} \
             ~{true="--remove-reference" false="" remove_reference} \
-            ~{true="--debug" false="" debug} \
+            --debug \
             --nthreads auto
         cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes | tee MAX_RAM
```

Comment:
While testing the WDL the "tee MAX_RAM" had some errors so I wrote to a file and caught the output in a "File" in the output section.  
cat /sys/fs/cgroup/memory/memory.max_usage_in_bytes > MAX_RAM
  
    output {
        ...
        ...
       	File  max_ram_usage_in_bytes = "MAX_RAM"
For reference here is the error from Terra with the original line of code:
<img width="413" alt="Screen Shot 2020-05-07 at 6 51 15 PM" src="https://user-images.githubusercontent.com/43828038/81352139-c94bc600-9093-11ea-91e2-95dba4b5e0d3.png">

Reply:
weird -- I wonder if the number is too big? Ok making the change, though I'm also separately dabbling with a more thorough heartbeat reporting mechanism.